### PR TITLE
Disown Feature

### PR DESCRIPTION
**Pull request by Ninja3047**
Added button to uploads page to disown/hide uploads. This sets the upload to hidden, but still retains the original uploader information in the database. Sorry for the extra commits...

### CODE REVIEW

File: srht/blueprints/api.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def upload():
             "shorthash": existing.shorthash,
             "url": file_link(existing.path)
         }
-    len = 4
+    len = 3
```
Comment (SirCmpwn) [inline srht/blueprints/api.py]: Why did you change this?

**Comment by Ninja3047** [comment]
Oh oops, I didn't mean to change that. How do I fix it?

**Comment by SirCmpwn** [comment]
I'll merge this as soon as I can test it - which won't be until Tuesday. Thanks!

**Comment by Ninja3047** [comment]
Ah okay, no problem.
